packages/sit-onyx/src/styles/grid.scss

```
@@ -167,4 +198,21 @@ $sidebar-breakpoints: (
       }
     }
   }
+
+  /**
+* Sets grid variables on root based on viewport size,
+* providing the properties for elements outside of container-queries e.g. popovers, dialogs 
+*/
+  @each $name, $maxVariant, $columns, $gutter, $margin in $gridVariants {
+    @if $name == "2xs" {
+      @include generate-grid-properties($columns, $gutter, $margin);
+    } @else {
+      @include screen(min, $name) {
+        @include generate-grid-properties($columns, $gutter, $margin);
+        @if $maxVariant != none {
+          @include generate-grid-max($name, $maxVariant);
+        }
+      }
+    }
+  }
```

We can skip `2xs` here, as it is mostly covered by the generated CSS for the container query (see the `2xs` condition). Only the root class needs to be added there.
```suggestion
    @each $name, $maxVariant, $columns, $gutter, $margin in $gridVariants {
    @if $name != "2xs" {
      @include screen(min, $name) {
        @include generate-grid-properties($columns, $gutter, $margin);
        @if $maxVariant != none {
          @include generate-grid-max($name, $maxVariant);
        }
      }
    }
  }
```